quakemigrate/io/data.py

```diff
@@ -126,20 +126,10 @@ def path_structure(self, archive_format="YEAR/JD/STATION", channels="*"):
         """
 
         if archive_format == "SeisComp3":
-            self.format = ("{year}/*/{station}/"+channels+"/*.{station}.*.*.D."
+            self.format = ("{year}/{network}/{station}/{channel}.D/{network}.{station}.{location}.{channel}.D."
                            "{year}.{jday:03d}")
-        elif archive_format == "YEAR/JD/*_STATION_*":
-            self.format = "{year}/{jday:03d}/*_{station}_*"
-        elif archive_format == "YEAR/JD/STATION":
-            self.format = "{year}/{jday:03d}/{station}*"
-        elif archive_format == "STATION.YEAR.JULIANDAY":
-            self.format = "*{station}.*.{year}.{jday:03d}"
-        elif archive_format == "/STATION/STATION.YearMonthDay":
-            self.format = "{station}/{station}.{year}{month:02d}{day:02d}"
-        elif archive_format == "YEAR_JD/STATION*":
-            self.format = "{year}_{jday:03d}/{station}*"
-        elif archive_format == "YEAR_JD/STATION_*":
-            self.format = "{year}_{jday:03d}/{station}_*"
```

Comment:
No need to hard deprecate these. Maybe add some logged statements here saying how to update the run scripts ahead of deprecation in the future.